packages/flutter/lib/src/gestures/events.dart

```diff
@@ -790,6 +790,61 @@ class PointerUpEvent extends PointerEvent {
   );
 }
 
+/// A base type for events that correspond to a discrete pointer signal.
```

Comment:
I wouldn't call this a "base" type, especially considering that it extends another type. :-)